basket/settings.py

```diff
@@ -144,10 +144,10 @@ def path(*args):
 MIDDLEWARE = (
     "allow_cidr.middleware.AllowCIDRMiddleware",
     "django.middleware.security.SecurityMiddleware",
+    "corsheaders.middleware.CorsMiddleware",
```

Comment:
Why move this here? Doesn't that mean that all static media will get CORS headers? Is that the intent?

Reply:
I can revert it back since we won't use it for the petitions. Good catch. But yeah, the docs say to move it before common middleware so I was testing it out.